components/label/src/index.js

```diff
@@ -48,6 +48,16 @@ const Label = props => <StyledLabel {...props} />;
 Label.propTypes = {
   /** Text for the label */
   children: PropTypes.node.isRequired,
+  /**
+   * Apply error state styling to the component
+   */
+  error: PropTypes.bool,
+  // NB these propTypes don't get documented :(
```

Comment:
should this be a TODO: or issue?

Reply:
it's a documentation issue, relating to many/most components

so yes - we should file an issue - not having documentation for the whitespace props is a bit of an oversight :)